pymc4/inference/utils.py

```diff
@@ -18,3 +19,20 @@ def initialize_state(model: Model, observed: Optional[dict] = None) -> flow.Samp
     """
     _, state = flow.evaluate_model_transformed(model, observed=observed)
     return state.as_sampling_state()
+
+def trace_to_arviz(pm4_trace):
+    """
+    Convert a PyMC4 trace as returned by sample() to an ArviZ trace object
+    that can be passed to e.g. arviz.plot_trace().
+
+    Parameters
+    ----------
+    pm4_trace : dict
+
+    Returns
+    -------
+    arviz.data.inference_data.InferenceData
+    """
+    import arviz as az
+    az_dict = {k: np.swapaxes(v.numpy(), 1, 0) for k, v in pm4_trace.items()}
```

Comment:
It could potentially be problematic if you have > 1d batch shape: https://github.com/arviz-devs/arviz/issues/456#issuecomment-528864981

But I guess in this case it is fine since we batch the log_prob ourself and the num_chain is only 1d

Reply:
That already works: `<xarray.DataArray 'hierarchical_model/beta' (chain: 50, draw: 200, hierarchical_model/beta_dim_0: 85)>`